sdk/src/main/java/io/radar/sdk/RadarApiClient.kt

```diff
@@ -102,6 +102,8 @@ internal class RadarApiClient(
             "X-Radar-Device-Model" to RadarUtils.deviceModel,
             "X-Radar-Device-OS" to RadarUtils.deviceOS,
             "X-Radar-Device-Type" to RadarUtils.deviceType,
+            "cf-ipcountry" to "CA",
+            "cf-region-code" to "MB",
```

Comment:
Hmm, curious what this is and interesting that we're hard coding these values (California and "MB"?). Mind helping me understand @tderouin ?